lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -514,11 +613,23 @@ public void shouldThrowOnNegativeNotBeforeLeeway() throws Exception {
                 .acceptNotBefore(-1);
     }
 
-// Issued At with future date
+    // Issued At with future date
+    @Test(expected = InvalidClaimException.class)
+    public void shouldThrowOnFutureIssuedAtDate() throws Exception {
+        Clock clock = mock(Clock.class);
+        when(clock.getNow()).thenReturn(Instant.ofEpochMilli(DATE_TOKEN_MS_VALUE - 1000));
+
+        String token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE0Nzc1OTJ9.CWq-6pUXl1bFg81vqOUZbZrheO2kUBd2Xr3FUZmvudE";
+        JWTVerifier.BaseVerification verification = (JWTVerifier.BaseVerification) JWTVerifier.init(Algorithm.HMAC256("secret"));
+
+        DecodedJWT jwt = verification.build(clock).verify(token);
+        assertThat(jwt, is(notNullValue()));
+    }
+
     @Test (expected = InvalidClaimException.class)
```

Comment:
These are the only 2 methods that use this way of testing expected exceptions. Note we already define a class-level field/property with the expected exception [here](https://github.com/auth0/java-jwt/blob/597fb2a5e817132143cbbec88c4e2ab4bc425843/lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java#L27). Can you please migrate this 2 to use the other approach?

Reply:
Updated.